pyscf/gto/basis/parse_cp2k_pp.py

```diff
@@ -24,7 +24,7 @@
 from pyscf.lib.exceptions import BasisNotFoundError
 import numpy as np
 
-def parse(string):
+def parse(string, symb=None):
```

Comment:
I don't think the `symb` keyword is used in that function. Is that there for consistency? Should individual element's PP be parsed?

Reply:
I guess this is just to make parse_cp2k_pp function signatures the same to that in parse_nwchem.

Reply:
Ok, thanks!

Reply:
I can also add symbol choosing for PPs, maybe later this week.